packages/ppx_poly_record/ppx_poly_record.1.0.1/descr

```diff
@@ -1,19 +1,31 @@
-ppx for polymorphic records
-Polymorphic record in OCaml
+ppx_pattern_guard: ppx extension for pattern guard
```

Comment:
The description (which is for `ppx_pattern_guard`) and the filename (which is for `ppx_poly_record`) don't match.